pkg/hubble/metrics/flow/handler.go

```diff
@@ -56,6 +56,7 @@ func (h *flowHandler) ProcessFlow(flow v1.Flow) {
 	switch flow.GetEventType().Type {
 	case monitorAPI.MessageTypeAgent:
 		typeName = "Agent"
+		subType = monitorAPI.AgentNotifications[monitorAPI.AgentNotification(flow.GetEventType().SubType)]
```

Comment:
this direct access worries me. can we guard?

Reply:
`monitorAPI.AgentNotifications` is a `map[AgentNotification]string`, so we'd just get an empty sub type for unsupported values. Or should we rather do something like:

```go
if s, ok := monitorAPI.AgentNotifications[monitorAPI.AgentNotification(flow.GetEventType().SubType)]; ok {
	subType = s
} else {
	subType = "unsupported"
}
```

Reply:
Huh. I don't know why I thought missing key would panic... Should be to leave as-is